fbw-a32nx/src/base/flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/A320_Neo_CDU_PerformancePage.js

```diff
@@ -1155,7 +1155,7 @@ class CDUPerformancePage {
         } else {
             const machAtManualCrossoverAlt = mcdu.casToMachManualCrossoverCurve.evaluate(selectedSpdMach)
             const manualCrossoverAltitude = mcdu.computeManualCrossoverAltitude(machAtManualCrossoverAlt);
-            const shouldShowMach = aircraftAltitude < manualCrossoverAltitude && (!mcdu._cruiseEntered || !mcdu.cruiseFlightLevel || manualCrossoverAltitude < mcdu.cruiseFlightLevel * 100);
+            const shouldShowMach = aircraftAltitude < manualCrossoverAltitude && (!mcdu.flightPlanService.getCruiseFlightLevel() || !mcdu.flightPlanService.getCruiseFlightLevel() || manualCrossoverAltitude < mcdu.flightPlanService.getCruiseFlightLevel() * 100);
```

Comment:
The same condition appears twice here I think?

Reply:
Ugh, yes I need to verify my logic is sufficient to replace `_cruiseEntered`. Thanks!